demo/src/lib/layout/Section.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+	import Svg from './Svg.svelte';
+	import link from 'bootstrap-icons/icons/link-45deg.svg?raw';
+
+	export let level: 1 | 2 | 3;
```

Comment:
We could (probably in another PR) compute the level automatically based on the context.

Reply:
Yes, it can be a good idea. To be investigated and discussed in another PR indeed.